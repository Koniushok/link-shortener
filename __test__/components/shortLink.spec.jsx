import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import ShortLink from '../../src/components/shortLink';

describe('InfoLink component', () => {
  const props = {
    link: 'testUrl',
  };

  describe('ShortLink component initial', () => {
    it('render correctly component', () => {
      const output = shallow(<ShortLink {...props} />);
      expect(shallowToJson(output)).toMatchSnapshot();
    });
  });
});
