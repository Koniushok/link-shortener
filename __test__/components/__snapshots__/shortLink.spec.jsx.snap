// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoLink component ShortLink component initial render correctly component 1`] = `
<div>
  <styled.a
    href="http://localhost:8080/testUrl"
    rel="noreferrer noopener"
    target="_blank"
  >
    http://localhost:8080/testUrl
  </styled.a>
  <CopyToClipboard
    text="http://localhost:8080/testUrl"
  >
    <styled.button>
      copy
    </styled.button>
  </CopyToClipboard>
</div>
`;
